backend/bin/deal-observer-backend.js

```diff
@@ -133,7 +133,14 @@ export const pieceIndexerLoop = async (makeRpcRequest, getDealPayloadCid, pgPool
     // Maximum number of deals to index in one loop iteration
     const maxDeals = 1000
     try {
-      await indexPieces(makeRpcRequest, getDealPayloadCid, pgPool, maxDeals)
+      const numOfMissingPayloadsCidsResolved = await indexPieces(makeRpcRequest, getDealPayloadCid, pgPool, maxDeals)
+      const numOfMissingPayloadsCids = await countStoredActiveDealsWithMissingPayloadCid(pgPool)
+      if (INFLUXDB_TOKEN) {
+        recordTelemetry('piece_indexer_stats', point => {
+          point.intField('total_missing_payload_cids', numOfMissingPayloadsCids)
+          point.intField('missing_payload_cids_resolved', numOfMissingPayloadsCidsResolved)
+        })
+      }
```

Comment:
I was trying to suggest more consistent naming, but then realized the existing naming needs to be improved first: https://github.com/CheckerNetwork/spark-deal-observer/pull/85

Reply:
To me it's confusing to work with a code base with naming that isn't obvious. Sometimes we call it "index pieces", sometimes we call it "resolve payload", sometimes "resolve payload cid". All mean very different things. After landing #85, I believe we can simplify this PR here too